.travis.yml

```diff
@@ -13,7 +13,7 @@ cache:
   - $HOME/.gradle/caches/
   - $HOME/.gradle/wrapper/
 script:
-  - ./gradlew check
+  - ./gradlew check -i
```

Comment:
what does the -i do?

Reply:
Yes it is. I will consider removing it. Was helpful in troubleshooting but produces awfully lot of logs.